src/main/java/dook/commands/DeleteCommand.java

```diff
@@ -0,0 +1,36 @@
+package dook.commands;
+
+import java.io.IOException;
+
+import dook.DookException;
+import dook.storage.Storage;
+import dook.tasks.Task;
+import dook.tasks.TaskList;
+import dook.ui.Ui;
+
+/**
+ * The command to delete a task from the TaskList.
+ */
+public class DeleteCommand extends Command {
+
+    private int taskNumber;
+
+    public DeleteCommand(int taskNumber) {
+        this.taskNumber = taskNumber;
+    }
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) throws DookException, IOException {
+        Task removed = tasks.delete(taskNumber - 1);
```

Comment:
Would it be better to rename the variable removed to removed task since variable names should be nouns?